### ISSUE TITLE
Add starcheck 13.5 

### ISSUE THREAD

**Issue by jeanconn** [sot/skare3#205]
'master' in starcheck is including https://github.com/sot/starcheck/pull/335 which has the minor changes to the launchers to have more reasonable warnings if the Perl pieces aren't present.  Additionally, the starcheck recipe has been updated to remove the Perl dependencies so a new conda build at any new version would be useful to remove the Perl tree.

**Comment by jeanconn** [comment]
And put it in ska3-flight

**Comment by taldcroft** [comment]
What's going on with this?

**Comment by jeanconn** [comment]
OK, so the "issue" system in skare3 had been suggested to me as "open an issue on a package when master has pulled away from the installed version so that won't get lost and we'll eventually cut a version".  That's true of this PR.  I thought we had decided the issues did not need full context for that approach.

In this case, 'master' in starcheck is including https://github.com/sot/starcheck/pull/335 which has the minor changes to the launchers to have more reasonable warnings if the Perl pieces aren't present.  Additionally, the starcheck recipe has been updated to remove the Perl dependencies so a new conda build at any new version would be useful to remove the Perl tree.  I had, however, thought that we would put in some more changes before cutting a starcheck release.
